pkg/skaffold/schema/v1alpha2/config.go

```diff
@@ -178,16 +178,12 @@ func defaultToDockerArtifacts(cfg *SkaffoldConfig) {
 	}
 }
 
-func setDefaultTagger(cfg *SkaffoldConfig, dev bool) {
+func setDefaultTagger(cfg *SkaffoldConfig) {
 	if cfg.Build.TagPolicy != (TagPolicy{}) {
 		return
 	}
 
-	if dev {
-		cfg.Build.TagPolicy = TagPolicy{ShaTagger: &ShaTagger{}}
-	} else {
-		cfg.Build.TagPolicy = TagPolicy{GitTagger: &GitTagger{}}
-	}
+	cfg.Build.TagPolicy = TagPolicy{GitTagger: &GitTagger{}}
```

Comment:
I'm guessing addresses #425 ?

Reply:
Yes! That's in the commit message. I've added that to the PR description. It greatly simplify the code and makes more sense for the users.